src/scripts/app.ts

```diff
@@ -132,57 +137,63 @@ function createToFindItem() {
 
 const checkElements = (event: Event) => {
 	const clickedImg = event.target as HTMLElement;
-	const imgFind = document.querySelector('.image-to-find') as HTMLImageElement;
 
+	const imgFind = document.querySelector('.image-to-find') as HTMLImageElement;
 	const clickedSrcValue = clickedImg.getAttribute('src');
 	const srcToFindValue = imgFind.getAttribute('src');
 	const clickedIdValue = clickedImg.getAttribute('id');
 	const idToFind = imgFind.getAttribute('id');
 
-	if (attemptCounter >= maxAttempts) {
-		showToast('error', 'Brak dalszych prób');
+	if (ATTEMPT_COUNTER >= MAX_ATTEMPTS) {
+		showToast('error', 'No further attempts, Try Again');
 		return;
 	}
 
-	if (clickedImg.classList.contains('js-game-img')) {
-		attemptCounter++;
+	if (clickedImg.classList.contains('game__image')) {
+		ATTEMPT_COUNTER++;
+		clickedImg.classList.remove('hidden');
 		console.log('clicked od the image', clickedImg);
-		checkMatch(clickedSrcValue, srcToFindValue, idToFind, clickedIdValue);
+
+		if (
+			(clickedSrcValue && srcToFindValue && idToFind && clickedIdValue) !== null
+		) {
+			checkMatchAndShowScoreboard(
+				clickedSrcValue,
+				srcToFindValue,
+				idToFind,
+				clickedIdValue
+			);
+		}
 	} else {
 		showToast('warning', 'You must choose an image');
 	}
 };
 
-const checkMatch = (
+const checkMatchAndShowScoreboard = (
 	clickedSrcValue: string | null,
 	srcToFindValue: string | null | undefined,
 	idToFind: string | null,
 	clickedIdValue: string | null
 ) => {
-	if (
-		clickedSrcValue !== null &&
-		srcToFindValue !== null &&
-		clickedIdValue !== null
-	) {
-		if (clickedSrcValue === srcToFindValue || idToFind === clickedSrcValue) {
-			playerScore++;
-			showScoreboard(computerScore, playerScore);
-			console.log('Match');
-		} else {
-			computerScore++;
-			showScoreboard(computerScore, playerScore);
+	if (clickedSrcValue === srcToFindValue || idToFind === clickedIdValue) {
+		PLAYER_SCORE++;
+		showScoreboard(COMPUTER_SCORE, PLAYER_SCORE, MAX_ATTEMPTS, ATTEMPT_COUNTER);
```

Comment:
Oprócz tego co pisałem wyżej, to generalnie nie jest to wymagane aby przekazywać te zmienne. One są dostępne globalnie, funkcja showScoreboard ma do nich dostęp bo są one poziom wyżej.

Dodatkowo, computer i player score + attempt counter powinny byc juz camelCase, bo je aktualizujemy, zmieniamy. Formatem MAX_ATTEMPS oznaczamy tylko stałe, ktore sie zupelnie nie zmieniaja.